pkg/netpol/connlist/connlist_test.go

```diff
@@ -218,6 +218,19 @@ func TestWithDotOutputFormat(t *testing.T) {
 	require.Equal(t, string(expectedOutput), dotRes)
 }
 
+func TestWithCsvOutputFormat(t *testing.T) {
+	dirPath := filepath.Join(testutils.GetTestsDir(), "onlineboutique_workloads")
+	analyzer := NewConnlistAnalyzer(WithOutputFormat("csv"))
+	res, err1 := analyzer.ConnlistFromDirPath(dirPath)
```

Comment:
might be ok to use `err` in all, with each new definition shadowing previous, instead of `err1`, `err2`, ...